### DISCUSSION TITLE
Stops working after ctrl shift q sign out

### DISCUSSION THREAD

**Discussion by tzhaki** [3kh0/ext-remover#81] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
No, it's not because chrome updated. It still works with other chromebooks. I have tried another chromebook after restarting it without ctrl shift q and it works just fine. This has happened on my chromebook and one of my friend's (both of which stopped working after being signed out via ctrl shift q). Have not tried hard resetting or draining battery yet, anyone got any ideas?

**Comment by phillipbruh** [comment, marked as answer]
use tampermonkey to auto do it

**Reply by tzhaki** [reply]
can't add extensions on our chromebooks

**Reply by phillipbruh** [reply]
make sure you have at least one other google account and then goto https://chrome.google.com/webstore?authuser=1

**Reply by ghost** [reply]
phillip i cant add other google accounts

**Reply by phillipbruh** [reply]
is it blocked?
